interface.go

```diff
@@ -0,0 +1,13 @@
+package ledger
+
+// SECP256K1 defines the necessary methods for ledger compatibility
```

Comment:
Can we replace this with the `Cosmos-SDK` type definition?
- https://github.com/cosmos/cosmos-sdk/blob/66dd2be12c181672068f12aa3643beeddd500843/crypto/ledger/ledger_secp256k1.go#L33

This was not always possible due to library conflicts, but I think it may be possible now.

If not, we should add a comment to link to the original.

Reply:
agree. Let's use that interface instead